content/guides/models/app/settings-page/billing-settings.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Manage your organization's billing settings
+menu:
+  default:
+    parent: settings
+title: Manage billing settings
+weight: 20
+---
+
+Navigate to your user profile page and select your user icon on the top right corner. From the dropdown, choose **Settings**, then click **Billing**.
+
+## Plan details
+
+### Teams plan
+The **Teams plan** section summarizes your organization's current plan, including a summary of monthly charges, as well as your current monthly usage:
+
+- Monthly charges.
+- Seats used. For details and a list of users, click **Manage users**.
+- Tracked hours used. For details, click **View usage**.
+- Amount of monthly data ingested by Weave. For details, click **View usage**.
+
+From here, you can compare plans or talk to Sales.
+
+### Storage
+The **Storage** section shows how much storage your organization uses, both free and paid. Each organization has 100 GB of free storage per month.
+
+To purchase additional storage, click **Buy storage**.
+
+To view details about each team's storage, click **Manage storage**. From there, you can optinally delete a team to free up its storage.
+
+## Plan usage
+This section visually summarizes the same usage information as in the **Team plan** section in a different way. For more details, click **View usage** on an individual tile. From here, you can also upgrade your Weave plan and manage storage.
+
+## Payment methods
+This section shows the payment methods on file for your organization. If you have not added a payment method, you will be prompted to do so when you upgrade your plan or add paid storage.
+
+## Billing admin
```

Comment:
QQ: Are we using "admin" or "administrator"?  (this is for my own knowledge as well).

Reply:
This is describing a UI that calls it *Billing admin**.